requests/__init__.py

```diff
@@ -70,6 +71,12 @@ def check_compatibility(urllib3_version, chardet_version):
     assert minor < 1
     assert patch >= 2
 
+# Check for older version of pyOpenSSL
+def __check_pyopenssl(pyopenssl_version, cryptography_version):
```

Comment:
This should really be _check_cryptography because that's what we're concerned about.